morebits.js

```diff
@@ -770,24 +781,43 @@ Morebits.quickForm.setElementLabel = function QuickFormSetElementLabel(element,
 	return true;
 };
 
+/**
+ * Stores the element's current label, and temporarily sets the label to the given text
+ * @param {(Element|Morebits.quickForm.element)} element
+ * @param {string} temporaryLabelText
+ */
 Morebits.quickForm.overrideElementLabel = function QuickFormOverrideElementLabel(element, temporaryLabelText) {
 	if (!element.hasAttribute("data-oldlabel")) {
 		element.setAttribute("data-oldlabel", Morebits.quickForm.getElementLabel(element));
 	}
 	return Morebits.quickForm.setElementLabel(element, temporaryLabelText);
 };
 
+/**
+ *  Restores the label stored by overrideElementLabel
+ * @param {(Element|Morebits.quickForm.element)} element
+ */
 Morebits.quickForm.resetElementLabel = function QuickFormResetElementLabel(element) {
 	if (element.hasAttribute("data-oldlabel")) {
 		return Morebits.quickForm.setElementLabel(element, element.getAttribute("data-oldlabel"));
 	}
 	return null;
 };
 
+/**
+ * Shows or hides a form element plus its label and tooltip
+ * @param {*} element
+ * @param {*} visibility
```

Comment:
I'm guessing `element` would have to either be of type `string` or `jQuery`? `visibility` I assume would be `boolean`?

Reply:
On testing, element can be HTMLElement or jQuery, visibility: boolean

Reply:
The code here would seem to allow for a string (the CSS selector). Just pointing this out; if only HTMLElement and jQuery makes sense here that's totally fine :)

Reply:
oh yeah, added it